rasa/core/test.py

```diff
@@ -144,24 +137,23 @@ class WronglyClassifiedUserUtterance(UserUttered):
 
     type_name = "wrong_utterance"
 
-    def __init__(
-        self,
-        text,
-        correct_intent,
-        correct_entities,
-        parse_data=None,
-        timestamp=None,
-        input_channel=None,
-        predicted_intent=None,
-        predicted_entities=None,
-    ):
-        self.predicted_intent = predicted_intent
-        self.predicted_entities = predicted_entities
+    def __init__(self, event: UserUttered, eval_store: EvaluationStore):
+
+        if eval_store.intent_predictions == list():
+            self.predicted_intent = None
+        else:
+            self.predicted_intent = eval_store.intent_predictions[0]
+        self.predicted_entities = eval_store.entity_predictions
 
-        intent = {"name": correct_intent}
+        intent = {"name": eval_store.intent_targets[0]}
```

Comment:
Are we sure that there is always at least one entry in `eval_store.intent_targets`?

Reply:
Yes, which comes from the `"true_intent"` parse data which is set in `core.training.dsl.EndToEndReader()`.  If the true intent is not present, a `ValueError` is thrown when reading the file.